example/unit-testing/README.md

```diff
@@ -38,7 +40,9 @@ The tests can be run by running:
 
 ```
 $ cd complex
-$ git clone https://github.com/ARMmbed/littlefs.git
+$ git clone -b v2.1.2 https://github.com/ARMmbed/littlefs.git
```

Comment:
`emubd` was renamed to `bd` in [`v2.2.0`](https://github.com/ARMmbed/littlefs/tree/v2.2.0), but wanted to stick with the exact version (by date) that I assume you used for the article.

Reply:
perfect, great idea